### ISSUE TITLE
connection.socket.send is not a function

### ISSUE THREAD

**Issue by Abhikumar98** [quirrel-dev/quirrel#66]
I've started the development server using concurrently as the steps were mentioned in the documentation.

But I'm getting this error and the dev server for quirrel keeps refreshing due to which a cron-job keeps getting added to the list.

`pages/api/cron.ts`

```import { CronJob } from "quirrel/vercel";
import fetch from "node-fetch";

export default CronJob("api/delete", "0 * * * *", async (job) => {
	try {
		console.log(job);
		const response = await fetch("/api/delete");
		const data = await response.json();

		console.log("data ---> ", data);
	} catch (error) {
		console.error(error);
	}
});
```

```This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
TypeError: connection.socket.send is not a function
    at /Users/abhishekkumar/Desktop/qrshare/node_modules/quirrel/dist/src/api/scheduler/routes/activity.js:68:43
    at JobsRepo.<anonymous> (/Users/abhishekkumar/Desktop/qrshare/node_modules/quirrel/dist/src/api/scheduler/jobs-repo.js:257:21)
    at step (/Users/abhishekkumar/Desktop/qrshare/node_modules/quirrel/dist/src/api/scheduler/jobs-repo.js:63:23)
    at Object.next (/Users/abhishekkumar/Desktop/qrshare/node_modules/quirrel/dist/src/api/scheduler/jobs-repo.js:44:53)
    at /Users/abhishekkumar/Desktop/qrshare/node_modules/quirrel/dist/src/api/scheduler/jobs-repo.js:38:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/abhishekkumar/Desktop/qrshare/node_modules/quirrel/dist/src/api/scheduler/jobs-repo.js:34:12)
    at Activity.onEvent (/Users/abhishekkumar/Desktop/qrshare/node_modules/quirrel/dist/src/api/scheduler/jobs-repo.js:253:74)
    at Activity.<anonymous> (/Users/abhishekkumar/Desktop/qrshare/node_modules/@quirrel/owl/dist/activity/activity.js:105:51)
    at step (/Users/abhishekkumar/Desktop/qrshare/node_modules/@quirrel/owl/dist/activity/activity.js:33:23)```

**Comment by Skn0tt** [comment]
I suspect this is a similar issue as https://github.com/quirrel-dev/quirrel/issues/65. I'm on it!

**Comment by Skn0tt** [comment]
This was fixed with v1.0.4: https://github.com/quirrel-dev/quirrel/issues/65#issuecomment-766621230
